### PR TITLE
Message list anna

### PR DESCRIPTION
**Pull request by ajgcph2019**
# Description

Please provide a short summary explaining what this PR is about.
I am working on : List of message components #9
I have made two components MessageList which should display the messages and Message .
I have also made a story for Message component.

Fixes # (issue)

# How to test?
npm run storybook

Please provide a short summary how your changes can be tested?

# Checklist

- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/App.js [hunk 1]
```diff
@@ -35,6 +36,7 @@ function App() {
           </AuthenticatedRoute>
         </Switch>
       </Router>
+      <MessageList />
```
Comment (benna100) [inline src/client/App.js:39]: Dont apply the component! Just make it. Its someone elses job to activate it

**Review by benna100 — CHANGES_REQUESTED** [review]
Look quite good anna, there are still a few things that need to be done.

Get it done asap. This is a very important story to version 1.0

File: src/client/App.js (hunk 1 above)
Comment (ajgcph2019) [reply src/client/App.js]: Done

File: src/client/components/Message/Message.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Messages.css';
+
+function Message({ username, text }) {
+  return (
+    <div>
```
Comment (tpetrina) [inline src/client/components/Message/Message.js:7]: Add class to the root called `chat-message`

File: src/client/components/Message/Message.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      <div className="chat_Body">
```
Comment (tpetrina) [inline src/client/components/Message/Message.js:8]: Why have two nested `div` elements?

File: src/client/components/Message/Message.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        <p className="chat_name"> {username}</p>
```
Comment (tpetrina) [inline src/client/components/Message/Message.js:9]: Can I suggest class names `chat-message__name` and `chat-message__message`?

**Review by tpetrina — COMMENTED** [review]
Really good work!
I would change the names of css class to be in the BEM format http://getbem.com/naming/. Look for comments below.

**Review by benna100 — COMMENTED** [review]
You have some empty files, if they are empty then dont commit them. 

There are also some comments from Toni

**Comment by younesmln** [comment]
Screenshot 2020-09-19 at 15 37 11
that's what I see in storybook

**Comment by ajgcph2019** [comment]
Working on this issue and will create another PR for merging

**Pull request closed without merging**